stdlib/experimental/experimental.flux

```diff
@@ -49,3 +49,6 @@ builtin window : (
   ?offset: duration,
   ?createEmpty: bool
 ) => [{T with _start: time, _stop: time, _time: time}]
+
+// An experimental version of integral.
+builtin integral : (<-tables: [{T with _time: time, _value: B}], ?unit: duration, ?interpolate: string) => [{T with _time: time, _value: B}]
```

Comment:
Since `integral` is an aggregate function, should it preserve the `_time` column? Or should it remove `_time`. The current behavior is to remove `_time`.

Reply:
Good catch, I think defaulting to the current behavior is probably the right move.